### PR TITLE
kat 2.0.1

### PR DESCRIPTION
**Pull request by maplesond**

### CODE REVIEW

**Comment by maplesond** [comment]
There are a couple of issues here.  First of all I need C++11 support.  How do I go about ensuring a suitable c++ compiler is available?  Second, I need to get the correct hashes for the bottles.

**Comment by dpo** [comment]
You can use `needs :cxx11` to request an appropriate compiler. You need not touch the `bottle` section of the formula. It will be automatically updated by the brew bot.

File: kat.rb [hunk 1]
```diff
@@ -17,23 +17,21 @@ class Kat < Formula
...
+  needs :cxx11
+
   depends_on "pkg-config" => :build
   depends_on "boost"
   depends_on "gnuplot"
-  depends_on "jellyfish-1.1"
-  depends_on "seqan"
 
   def install
     ENV.libstdcxx if ENV.compiler == :clang && MacOS.version >= :mavericks
```
Comment (dpo) [inline kat.rb]: Should be `libcxx` instead of `libstdcxx`.

Comment (dpo) [inline kat.rb]: next build error:
```
jellyfish_helper.cc:47:10: fatal error: 'bits/stl_vector.h' file not found
```

**Comment by maplesond** [comment]
I suspect the reason 'bits/stl_vector.h' is not present is due to an older than expected version of GCC on the test systems.  I oldest version I've tested on is 4.9.1.  So it would be good add some additional requirements for the C++ libraries and/or compiler versions.  I'm not sure how to do that in homebrew however.

**Comment by dpo** [comment]
I see. You need a `fails_with` block. Grepping through existing formulas should show you how it's used.

**Comment by maplesond** [comment]
Need to modify source code to make it more portable for C++11 compilers on mac.  Will close this case and reopen a new one targetting the new version of KAT.

**Pull request closed without merging**